tokio/src/task/join_set.rs

```diff
@@ -23,7 +23,10 @@ use crate::util::IdleNotifiedSet;
 ///
 /// All of the tasks must have the same return type `T`.
 ///
-/// When the `JoinSet` is dropped, all tasks in the `JoinSet` are immediately aborted.
+/// When the `JoinSet` is dropped, all non-blocking tasks in the `JoinSet` are
+/// immediately aborted. Tasks spawned with
+/// [`spawn_blocking`](Self::spawn_blocking) will abort when they reach an
+/// `await` point
```

Comment:
It doesn't make sense to talk about `.await` points with `spawn_blocking`.

You may want to reword these things by referencing [the general docs on cancellation of tasks](https://docs.rs/tokio/latest/tokio/task/index.html#cancellation), and by copying the note about `spawn_blocking` from there into the right spots.

Reply:
> It doesn't make sense to talk about `.await` points with `spawn_blocking`.

Ahh yeah ofc, I should have taken the time to look at the definition of spawn_blocking :sweat_smile:.

> by copying the note about spawn_blocking from there into the right spots.

Thats perfect!